openapi.yaml

```diff
@@ -91,3 +144,116 @@ components:
           example: JWT
           type: string
           enum: [JWT]
+    TopicsGetResponse:
+      type: array
+      items:
+        type: array
+        items:
+          $ref: '#/components/schemas/TopicInfo'
+    TopicInfo:
+      type: object
+      additionalProperties: false
+      properties:
+        topicid:
+          title: Topic identifier
+          description: This identifier is used in Klaw metadata store to ensure uniquenes.
+          type: integer
+          format: int32
+          example: 1010
+        totalNoPages:
+          title: Total number of pages
+          type: string
+          example: "1"
+        currentPage:
+          title: Current page number
+          type: string
+          example: "1"
+        allPageNos:
+          title: All page numbers
+          description: List of all page numbers
+          type: array
+          items:
+            type: string
+          example: ["1"]
+        topicName:
+          title: Topic name
+          description: Kafka Topic name
+          type: string
+          example: "topicName"
+        noOfPartitions:
+          title: Number of partitions
+          description: Topic partition count
+          type: integer
+          format: int32
+          example: 10
+        description:
+          title: Description
+          description: Kafka topic description stored in Klaw metadata.
+          type: string
+          example: "Topic description"
```

Comment:
Maybe we can use another example here? As far as I see, "Topic description" is kind of a placeholder and will always been set if there is no actual description. Which does not need to be reflected in the doc here, but I still think a different example here would make that more clear.